### PR TITLE
chore: Backport 10680 to release 4.4

### PR DESCRIPTION
**Pull request by 0x656b694d**
Co-authored-by: Ross Tannenbaum <rtannenb@redhat.com>
Co-authored-by: J. Victor Martins <jvdm@sdf.org>
(cherry picked from commit 9eccafc5189ec7726b5b189fe2b561616820c37a)

### CODE REVIEW

**Comment by daynewlee** [comment]
We need to use origin/master pkg/registries/types/types.go 
https://github.com/stackrox/stackrox/pull/11095/commits/16f5e2cde2970711affbc5a2c347bf1d44425bdb
That should do the trick

**Comment by RTann** [comment]
Interesting why golangci-lint is complaining...?

**Review by RTann — COMMENTED** [review]
LGTM but not sure why golangci-lint suddenly has an issue with this

File: scanner/indexer/options.go [hunk 1]
```diff
@@ -26,6 +27,12 @@ func WithAuth(auth authn.Authenticator) Option {
 	}
 }
 
+func InsecureSkipTLSVerify(insecure bool) Option {
```
Comment (RTann) [inline scanner/indexer/options.go:30]: I guess you can say "InsecureSkipTLSVerify" specifies if TLS verification should be skipped when reaching out to the registry.

File: pkg/scannerv4/client/options.go [hunk 2]
```diff
@@ -42,6 +42,10 @@ type options struct {
 	comboMode   bool
 }
 
+type ImageRegistryOpt struct {
```
Comment (RTann) [inline pkg/scannerv4/client/options.go:45]: "ImageRegistryOpt" defines options for reaching out to image registries.

**Review by daynewlee — APPROVED** [review]

**Pull request merged**